src/api/Jalasoft.TeamUp.Projects.API/Controllers/GetProjects.cs

```diff
@@ -0,0 +1,32 @@
+namespace Jalasoft.TeamUp.Projects.API.Controllers
+{
+    using System.Net;
+    using Jalasoft.TeamUp.Projects.Core.Interfaces;
+    using Jalasoft.TeamUp.Projects.Models;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Azure.WebJobs.Extensions.Http;
+    using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+    using Microsoft.OpenApi.Models;
+
+    public class GetProjects
+    {
+        private readonly IProjectsService projectService;
+
+        public GetProjects(IProjectsService projectService)
+        {
+            this.projectService = projectService;
+        }
+
+        [FunctionName("projects")]
+        [OpenApiOperation(operationId: "Run", tags: new[] { "Projects" })]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(Project), Description = "Successful response")]
+        public IActionResult Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req)
```

Comment:
Change null to "v1/projects"

Reply:
Has change route

![Capture](https://user-images.githubusercontent.com/85975279/126638407-8400df17-01cd-4b32-abe4-acabe6bffe24.PNG)